src/ProjectOrigin.Vault/Options/JobOptions.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace ProjectOrigin.Vault.Options;
+public class JobOptions : IValidatableObject
+{
+    public const string Job = nameof(Job);
+
+    [Required]
+    public int CheckForWithdrawnCertificatesIntervalInSeconds { get; set; }
+    public int TimeBeforeItIsOkToRunCheckForWithdrawnCertificatesAgain()
+    {
+        return ((CheckForWithdrawnCertificatesIntervalInSeconds * 2) / 3);
```

Comment:
Not entirely sure why we * 2 / 3? Is there a reason.

Reply:
I just needed some time for when the worker is being considered to have 'just run'. Thought 2/3 of the time was fitting